Resources/Prototypes/Floof/Interactions/base.yml

```diff
@@ -0,0 +1,29 @@
+# Base interaction that involves hands
```

Comment:
This entire file can be removed in favor of the suggestion below

Reply:
will look into this when i have some time,  been a bit busty and have not had time to look into this yet.  sorry for the delay